include/deal.II/matrix_free/mapping_info_storage.templates.h

```diff
@@ -135,7 +136,10 @@ namespace internal
       // Jacobians (these two together will give use the gradients of the
       // inverse Jacobians, which is what we need)
       if ((update_flags & update_hessians) != 0u ||
-          (update_flags & update_jacobian_grads) != 0u)
+          (update_flags & update_jacobian_grads) != 0u ||
```

Comment:
Do we still have these (`!= 0u`)?

Reply:
In quite a few places in `include/matrix_free/evaluation_kernels.h` as well.